flux-middle/src/fhir.rs

```diff
@@ -56,14 +56,15 @@ pub struct Qualifier {
 /// note: `Map` is a very generic name, so we typically use the type qualified as `fhir::Map`.
 #[derive(Default, Debug)]
 pub struct Map {
-    uifs: FxHashMap<Symbol, UifDef>,
-    consts: FxHashMap<Symbol, ConstInfo>,
-    qualifiers: Vec<Qualifier>,
-    adts: FxHashMap<LocalDefId, AdtDef>,
-    structs: FxHashMap<LocalDefId, StructDef>,
-    enums: FxHashMap<LocalDefId, EnumDef>,
-    fns: FxHashMap<LocalDefId, FnSig>,
-    assumes: FxHashSet<LocalDefId>,
+    pub(crate) uifs: FxHashMap<Symbol, UifDef>,
+    pub(crate) defns: FxHashMap<Symbol, Defn>,
+    pub(crate) consts: FxHashMap<Symbol, ConstInfo>,
+    pub(crate) qualifiers: Vec<Qualifier>,
+    pub(crate) adts: FxHashMap<LocalDefId, AdtDef>,
+    pub(crate) structs: FxHashMap<LocalDefId, StructDef>,
+    pub(crate) enums: FxHashMap<LocalDefId, EnumDef>,
+    pub(crate) fns: FxHashMap<LocalDefId, FnSig>,
+    pub(crate) assumes: FxHashSet<LocalDefId>,
```

Comment:
I think we don't need to expose these anymore

```suggestion
    uifs: FxHashMap<Symbol, UifDef>,
    defns: FxHashMap<Symbol, Defn>,
    consts: FxHashMap<Symbol, ConstInfo>,
    qualifiers: Vec<Qualifier>,
    adts: FxHashMap<LocalDefId, AdtDef>,
    structs: FxHashMap<LocalDefId, StructDef>,
    enums: FxHashMap<LocalDefId, EnumDef>,
    fns: FxHashMap<LocalDefId, FnSig>,
    assumes: FxHashSet<LocalDefId>,
```